opencti-platform/opencti-front/src/private/components/workspaces/WorkspaceHeader.jsx

```diff
@@ -397,6 +492,103 @@ const WorkspaceHeader = ({
           </Slide>
         </Security>
       </div>
+      <Dialog
```

Comment:
what do you think about moving all this dialog to a separate component ? along with all the containers search logic.

Reply:
I'm working on it

Reply:
I agree. Did it on last edit